testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -35,6 +35,7 @@ impl SwarmDirectory {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
Why not just delete the function?

Reply:
Is this annotation still needed? Lets remove it or make sure that the SwarmDirectory type is made public